.env

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
docker-compose automatically looks for .env

require('dotenv').config();
Automatically knows to load the .env file